app/services/api/get_trainees_service.rb

```diff
@@ -8,13 +8,17 @@ def initialize(provider:, params: {})
     end
 
     def call
-      provider.trainees
-              .joins(:start_academic_cycle)
-              .where(academic_cycles: { id: academic_cycle.id })
-              .where("trainees.updated_at > ?", since)
-              .order("trainees.updated_at #{sort_by}")
-              .page(page)
-              .per(pagination_per_page)
+      trainees = provider.trainees
```

Comment:
Any chance we can reuse this
https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/services/trainees/filter.rb


This is so that we don't end up forking/reinventing search...